packages/utils/src/browser.ts

```diff
@@ -1,10 +1,8 @@
 import { isString } from './is';
-import { GLOBAL_OBJ } from './worldwide';
+import { getGlobalObject } from './worldwide';
 
-/**
- * TODO: Move me to @sentry/browser when @sentry/utils no longer contains any browser code
- */
-export const WINDOW = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
+// eslint-disable-next-line deprecation/deprecation
+const WINDOW = getGlobalObject<Window>();
```

Comment:
I think this still causes issues right? Since `Window` is at the top level.

Reply:
I don't think the user reported issue has anything to do with top-level `Window` usage since the error directly references `browser.d.ts`:
```
node_modules/@sentry/utils/types/browser.d.ts:4:69 - error TS2304: Cannot find name 'Window'.
```

In `master` there is `Window` in (`browser.d.ts`):
```ts
export declare const WINDOW: InternalGlobal & Window;
export declare function htmlTreeAsString(elem: unknown, keyAttrs?: string[]): string;
export declare function getLocationHref(): string;
export declare function getDomElement<E = any>(selector: string): E | null;
```

After this PR changes, `browser.d.ts` no longer has this:
```ts
export declare function htmlTreeAsString(elem: unknown, keyAttrs?: string[]): string;
export declare function getLocationHref(): string;
export declare function getDomElement<E = any>(selector: string): E | null;
```
And in the outputted JavaScript there are no generics and `Window` is gone too:
```ts
const WINDOW = GLOBAL_OBJ ;
```

Reply:
Ah of course, it's not exported!